op-chain-ops/genesis/config.go

```diff
@@ -610,7 +598,7 @@ func (d *DeployConfig) RollupConfig(l1StartBlock *types.Block, l2GenesisBlockHas
 			L2Time: l1StartBlock.Time(),
 			SystemConfig: eth.SystemConfig{
 				BatcherAddr: d.BatchSenderAddress,
-				Overhead:    eth.Bytes32(common.BigToHash(new(big.Int).SetUint64(d.GasPriceOracleOverhead))),
+				Overhead:    eth.Bytes32{},
```

Comment:
This is worth getting the eyes of the superchain registry maintainers on, cc @geoknee. The idea is that now there is no need to track the overhead and its always 0 for chains that spin up with ecotone at genesis

Reply:
I have reverted this change, no need to look into now